### ISSUE TITLE
My transactions doenst show on solana explorer

### ISSUE THREAD

**Issue by aupz** [simplyjarod/solana-validator#2]
I configured a machine for devnet and run the script, everything goes smooth and i can call some rpc methos on my subdomain, but if a try to mint something on devnet  for exemple, the transaction never goes. Always fail and never show on solana explorer. am I doing something wrong?

**Comment by simplyjarod** [comment]
How are you minting?
If you execute `solana catchup ~/validator-key.json` you got the "caughtup" message?
If you execute `solana gossip | grep yo.u.r.ip` can you see your host with open ports and solana version?

**Comment by aupz** [comment]
Thx for your reply! It doenst show caughtup, i'm always failing behind, im using google cloud instance n2-standard-64. (64 vcpu and 256 gb ram) 
Is the problem my machine specs? or some configuration that i did?

**Comment by simplyjarod** [comment]
That CPU and RAM is good enough. What disks are you using and what setup? Hope there are more than 1 and you use a ramdisk too.
After a big time of failing behind, you will never gain slots. Reboot your server, launch solana validator and wait for gossip first and catchup after it (steps 3, 4 and 5 from https://github.com/simplyjarod/solana-validator/discussions/1). 
Be patient, you can go for a walk with your dog in the meantime.
Of course, double check your logs to be sure nothing goes wrong. Hint: have you got your ports open?

**Comment by aupz** [comment]
Yep ports are opened, i used two disks nvme 1.5tera each, followed up the system tunning on solana docs. https://vastorigins.com/2022/03/18/create-solana-validator-rpc-only-node-part-1/ i used this tutorial for disk setup, i dont know if is something wrong with it.  I will try on a bare metal server this time, I'm negotiating with maxihost, i will use your script again and see what happen. Do you know if theres a place where i can see what each args for solana-validator does ?

**Comment by simplyjarod** [comment]
`solana-validator --help` will list and describe all args
